pkg/testutil/testutil.go

```diff
@@ -173,6 +173,8 @@ func TolerantVerifyLeakMain(m *testing.M) {
 		// https://github.com/kubernetes/klog/blob/c85d02d1c76a9ebafa81eb6d35c980734f2c4727/klog.go#L417
 		goleak.IgnoreTopFunction("k8s.io/klog/v2.(*loggingT).flushDaemon"),
 		goleak.IgnoreTopFunction("k8s.io/klog.(*loggingT).flushDaemon"),
+		// https://github.com/baidubce/bce-sdk-go/blob/9a8c1139e6a3ad23080b9b8c51dec88df8ce3cda/util/log/logger.go#L359
```

Comment:
Can we add issue for this on github.com/baidubce/bce-sdk-go?

Reply:
I have reported this issue with our internal colleagues, and I will continue to follow up the SDK update.

Reply:
💪🏽